angular.json

```diff
@@ -100,6 +100,14 @@
               "**/node_modules/**"
             ]
           }
+        },
+        "server": {
```

Comment:
tell angular cli to build server using respective entry points, output paths and compiler configurations etc